src/main/java/com/health/utility/ServiceUtility.java

```diff
@@ -393,60 +395,193 @@ public static boolean checkCatAndAllImageFile(MultipartFile temp) {
+
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyyMMdd'T'HHmmss");
+        String zipFileName = "scripts-" + sdf.format(new Date()) + ".zip";
+
+        Path zipFilePath = Paths.get(env.getProperty("spring.applicationexternalPath.name"),
+                CommonData.uploadDirectoryScriptZipFiles, zipFileName);
+
+        try (OutputStream fos = Files.newOutputStream(zipFilePath); ZipOutputStream zos = new ZipOutputStream(fos)) {
+
+            byte[] buffer = new byte[16384];
 
-            Elements videoTags = doc.select("video");
-            for (Element video : videoTags) {
-                video.remove();
+            for (String fileUrl : fileUrls) {
+                Path fileUrlPath = Paths.get(env.getProperty("spring.applicationexternalPath.name"), fileUrl);
+
+                File file = fileUrlPath.toFile();
+
+                if (!file.exists()) {
+                    logger.info("Sciptfile not found: {}" + file);
+                    continue;
+                }
+
+                try (FileInputStream fis = new FileInputStream(file)) {
+                    zos.putNextEntry(new ZipEntry(file.getName()));
+
+                    int length;
+                    while ((length = fis.read(buffer)) > 0) {
+                        zos.write(buffer, 0, length);
+                    }
+
+                    zos.closeEntry();
+                }
             }
 
-            Elements sourceTags = doc.select("source");
-            for (Element source : sourceTags) {
-                source.remove();
+            String temp = zipFilePath.toString();
+            int indexToStart = temp.indexOf("Media");
+            document = temp.substring(indexToStart, temp.length());
+
+        } catch (Exception e) {
+            logger.error("Exception Error", e);
+        }
+
+        return document;
+    }
+
+    /****
+     * Create ZipFileWithSubDirectories
+     * 
+     */
+
+    private static void zipFile(File fileToZip, String fileName, ZipOutputStream zipOut) throws IOException {
+        if (fileToZip.isHidden()) {
+            return;
+        }
+        if (fileToZip.isDirectory()) {
+            if (fileName.endsWith("/")) {
+                zipOut.putNextEntry(new ZipEntry(fileName));
+                zipOut.closeEntry();
+            } else {
+                zipOut.putNextEntry(new ZipEntry(fileName + "/"));
+                zipOut.closeEntry();
+            }
+            File[] children = fileToZip.listFiles();
+            for (File childFile : children) {
+                zipFile(childFile, fileName + "/" + childFile.getName(), zipOut);
             }
+            return;
+        }
+        FileInputStream fis = new FileInputStream(fileToZip);
+        ZipEntry zipEntry = new ZipEntry(fileName);
+        zipOut.putNextEntry(zipEntry);
+        byte[] bytes = new byte[16384];
+        int length;
+        while ((length = fis.read(bytes)) >= 0) {
+            zipOut.write(bytes, 0, length);
+        }
+        fis.close();
```

Comment:
```suggestion
        }
```
close not required with try-with-resources

close the brace instead